client/src/ui/Review.jsx

```diff
@@ -191,7 +199,7 @@ export default class Review extends Component {
                       alt={this.state.liked ? "Liked" : "Not Liked Yet"}
                     />
                     <p className="upvote-text">Helpful
-                        ({this.state.numLikes})</p>
+                        ({this.state.numLikes ? this.state.numLikes : 0})</p>
```

Comment:
I just recently found out about the "??" operator. a ?? b = b if a is null/undefined. Useful for things like this.